<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFileDialog</class>
 <widget class="QDialog" name="CreateFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>462</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,5,10,1">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文件名:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="0" column="2" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>260</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>路径：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="pathEdit"/>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="filleBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
